clang-tools-extra/clangd/test/type-hierarchy.test

```diff
@@ -88,56 +89,6 @@
 # CHECK-NEXT:     }
 # CHECK-NEXT:  ]
 ---
-{"jsonrpc":"2.0","id":2,"method":"typeHierarchy/subtypes","params":{"item":{"uri":"test:///main.cpp","data":{"parents":[{"parents":[{"parents":[],"symbolID":"FE546E7B648D69A7"}],"symbolID":"ECDC0C46D75120F4"}],"symbolID":"8A991335E4E67D08"},"name":"Child2","kind":23,"range":{"end":{"character":13,"line":3},"start":{"character":7,"line":3}},"selectionRange":{"end":{"character":13,"line":3},"start":{"character":7,"line":3}}}}}
```

Comment:
What was the reasoning for removing this entirely?

Reply:
Huh; I don't recall. The test case seems to still be valid, so I restored it.